rma/views/rma_order_line_view.xml

```diff
@@ -0,0 +1,539 @@
+                                string="Deliveries"
+                            />
+                        </button>
+                        <!--Move
+                        this button to rma_account-->
+                        <!--<button
+                        type="object"  name="action_view_invoice"
+                                    class="oe_stat_button"
+                                    icon="fa-pencil-square-o"
+                                    string="Origin Inv">
+                            </button>-->
+                        <button
+                            type="object"
+                            name="action_view_rma_lines"
+                            class="oe_stat_button"
+                            icon="fa-link"
+                            groups="stock.group_stock_user"
+                            invisible="rma_line_count == 0"
+                        >
+                            <field
+                                name="rma_line_count"
+                                widget="statinfo"
+                                string="Supplier RMA"
+                            />
+                        </button>
+                    </div>
+                    <div class="oe_title" name="title">
+                        <h1>
+                            <field
+                                name="name"
+                                readonly="state not in ['draft']"
+                                invisible="context.get('hide_title',False)"
+                            />
+                        </h1>
+                    </div>
+                    <group>
+                        <group name="partner">
+                            <field
+                                name="partner_id"
+                                context="{'res_partner_search_mode': 'customer'}"
+                                string="Customer"
+                                readonly="state not in ['draft']"
+                            />
+                            <field
+                                name="delivery_address_id"
+                                domain="['|', ('parent_id', '=', partner_id),
+                                                         ('id', '=', partner_id)]"
+                                context="{'show_address': 1}"
+                                options="{'always_reload': 1}"
+                                groups='rma.group_rma_delivery_invoice_address'
+                                readonly="state not in ['draft']"
+                            />
+                        </group>
+                        <group>
+                            <group name="contact">
+                                <field name="date_rma" />
+                                <field name="requested_by" readonly="1" />
+                                <field name="assigned_to" />
+                                <field name="type" invisible="1" />
```

Comment:
I think having this field with invisible here only to be used in attributes is not necessary anymore

Reply:
Done, Thanks